codespell_lib/data/dictionary.txt

```diff
@@ -27800,6 +27806,8 @@ renegoziator->renegotiator
 renegoziators->renegotiators
 reneweal->renewal
 renewl->renewal
+renforce->reinforce
+renforced->reinforced
```

Comment:
`renforce` should be French?
```text
renforcement->reinforcement
renforcements->reinforcements
renforces->reinforces
```
* https://github.com/search?q=renforcement&type=code
* https://github.com/search?q=renforcements&type=code
* https://github.com/search?q=renforces&type=code

Reply:
Yes, but sometimes french words make it in to english source. So i think it's valid